lib/logger/lib/logger.ex

```diff
@@ -275,12 +275,8 @@ defmodule Logger do
   The console backend allows you to customize the format of your log messages
   with the `:format` option.
 
-  You may set `:format` to either a string or a `{module, function}` tuple if
-  you wish to provide your own format function. The `{module, function}` will be
-  invoked with the log level, the message, the current timestamp and the
-  metadata.
-
-  Here is an example of how to configure the `:console` backend in a
+  You may set `:format` to either a string or a `{module, function}` tuple if you wish to provide
+  your own format function. Here is an example of how to configure the `:console` backend in a
```

Comment:
Cosmetic: should we keep around the same line length as previous paragraphs?

Reply:
Yes, fixing it.